catalog-info.yaml

```
@@ -1,18 +1,125 @@
 apiVersion: backstage.io/v1alpha1
-manifestVersion: 0.1.1
+kind: System
+metadata:
+  name:  boleto-api
+  title: boleto-api
+  namespace: mundipagg
+  description: Gateway de registro online de transações do meio de pagamento de boleto.
+  links:
+    - url: https://mundipagg.atlassian.net/wiki/spaces/PROC/pages/5650448827/Boleto+-+Overview
+      title: Documentação técnica
+      icon: dashboard
+  annotations:
+    github.com/project-slug: mundipagg/boleto-api
+    pagar.me/ci: azure-devops
+    pagar.me/cd: azure-devops
+    stone.co/sox: "true"
+    stone.co/pii: "true"
+spec:
+  type: backend
+  owner: pagarme/processing-banks
+  lifecycle: production
+---
+apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
   name: boleto-api
-  description: Gateway de registro online de transações do meio de pagamento de boleto
-  compliance:
-    gui: false
-    sox: sox
-    pci: none
+  namespace: mundipagg
+  title: boleto-api-server
+  description: Gateway de registro online de transações do meio de pagamento de boleto.
+  links:
+    - url: https://mundipagg.atlassian.net/wiki/spaces/PROC/pages/5650448827/Boleto+-+Overview
+      title: Documentação técnica
+      icon: dashboard
+  annotations:
+    github.com/project-slug: mundipagg/boleto-api
+    pagar.me/ci: azure-devops
+    pagar.me/cd: azure-devops
+    stone.co/sox: "true"
+    stone.co/pii: "true"
 spec:
   lifecycle: production
-  type: service
-  owner: processing.banks@pagar.me
+  type: backend
+  owner: pagarme/processing-banks
   monitoring:
     platform: newrelic
-    owner: processing.banks@pagar.me
-    alert_name: synthetic_mundipagg_boletoapi
+    owner: pagarme/processing-banks
+---
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: mongo
+  title: Boleto
+  description: Banco de dados da boleto-api
+  annotations:
+    stone.co/pci: "true"
+    stone.co/sox: "true"
+    stone.co/pii: "true"
+spec:
+  type: database
+  owner: pagarme/processing-banks
+  system: mundipagg/boleto-api
+  lifecycle: production
+---
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: RabbitMQ
+  description: Fila de resiliência de registro de boletos
+  annotations:
+    stone.co/pci: "true"
+    stone.co/sox: "true"
+    stone.co/pii: "true"
+spec:
+  type: queue
+  owner: pagarme/processing-banks
+  system: mundipagg/boleto-api
+  lifecycle: production
+---
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: Blob
+  description: Storage de armazenamento de certificados.
```

Algo nessa linha eu acho 😀 
```suggestion
  description: Storage de armazenamento de certificados e segundo nível de resiliência de registro de boletos.
```